### PR TITLE
Update thunderbird_privacy_policy.md

### PR DESCRIPTION
**Pull request by mafeld**
Calendar and Appointment update

### CODE REVIEW

**Review by peiying2 — APPROVED** [review]

File: en/thunderbird_privacy_policy.md [hunk 1]
```diff
@@ -27,6 +27,24 @@ __Email domain__: Thunderbird receives your email address domain. Your full emai
 
 __Technical data__: Thunderbird also receives information about the application’s version and device operating system. When Thunderbird sends technical data to us, your IP address is temporarily collected as part of our server logs.
 
+### Set-Up and Configure Your Calendar
```
Comment (flodolo) [inline en/thunderbird_privacy_policy.md:30]:
```suggestion
### Set Up and Configure Your Calendar
```

File: en/thunderbird_privacy_policy.md [hunk 2]
```diff
@@ -27,6 +27,24 @@ __Email domain__: Thunderbird receives your email address domain. Your full emai
...
+
+Thunderbird collects the domain for your email/calendar, as well as technical data to set-up and configure your calendar. Other information, like your name, your calendar events, and event attendees are stored locally on your computer and never sent to us.
```
Comment (flodolo) [inline en/thunderbird_privacy_policy.md:32]:
```suggestion
Thunderbird collects the domain for your email/calendar, as well as technical data to set up and configure your calendar. Other information, like your name, your calendar events, and event attendees are stored locally on your computer and never sent to us.
```

File: en/thunderbird_privacy_policy.md [hunk 3]
```diff
@@ -27,6 +27,24 @@ __Email domain__: Thunderbird receives your email address domain. Your full emai
...
+
+You can connect your Google, Microsoft, or Apple calendar to Thunderbird and control your calendar right from Thunderbird. 
+
+We do not store your content from integrating Apple Calendar, Google Calendar, or Microsoft 365 with Thunderbird. That information stays on your computer. Nor do we share your content with any third party other than your calendar provider and anyone you specifically choose to send the appointments to.
+
+### Set-Up and Schedule Calendar Appointments with Thunderbird Appointment
```
Comment (flodolo) [inline en/thunderbird_privacy_policy.md:38]:
```suggestion
### Set Up and Schedule Calendar Appointments with Thunderbird Appointment
```

**Review by flodolo — COMMENTED** [review]
I believe all the "set-up" (noun, equivalent to setup) should be "set up" (verb).

**Review by flodolo — APPROVED** [review]

**Pull request merged**